website/docs/r/wafv2_web_acl_logging_configuration.html.markdown

```diff
@@ -77,7 +77,7 @@ resource "aws_wafv2_web_acl_logging_configuration" "example" {
 
 The following arguments are supported:
 
-* `log_destination_configs` - (Required) The Amazon Kinesis Data Firehose Amazon Resource Name (ARNs) that you want to associate with the web ACL. Currently, only 1 ARN is supported.
+* `log_destination_configs` - (Required) The Amazon Kinesis Data Firehose Amazon Resource Name (ARNs), S3 or Cloudwatch Logs groups(ARNs) that you want to associate with the web ACL. 
```

Comment:
Tweaked the phrasing slightly to make it a bit more streamlined.

```suggestion
* `log_destination_configs` - (Required) The Amazon Kinesis Data Firehose, Cloudwatch Log log group, or S3 bucket Amazon Resource Names (ARNs) that you want to associate with the web ACL. 
```

Reply:
@justinretzolk  Thanks for your update and this looks good.